tutor/hooks/consts.py

```diff
@@ -113,19 +124,23 @@ def your_filter(items):
     #:     - ``path`` is a tuple that corresponds to a template relative path.
     #:       Example: ``("myplugin", "hooks", "myservice", "pre-init")`` (see:py:data:`IMAGES_BUILD`).
     #:       The command to execute will be read from that template, after it is rendered.
-    COMMANDS_INIT = filters.get("commands:init")
+    COMMANDS_INIT: Filter[list[tuple[str, tuple[str, ...]]], str] = filters.get(
```

Comment:
Subscripting the built-ins reads much better... @regisb are you interested in having this change be made across the codebase? If so I can create an issue for it. It'd be a `good-first-issue`.

Reply:
Yes, absolutely.